docs/reference/why_pmm.md

```diff
@@ -19,17 +19,10 @@
     - [Setting up PMM Server](setting-up/server/index.md)
     - [Setting up PMM Client](setting-up/client/index.md)
 
-## How can I upgrade from version 1?
+## How can I upgrade from version 2?
 
-There is no direct software upgrade path.
+There is no direct software upgrade path. For infomation, see [Upgrade from PMM2](../pmm-upgrade/upgrade_from_pmm_2.md).
```

Comment:
the same here

Reply:
looks like another duplicate topic probably following the restructure. I have deleted it for now and created [a task](https://perconadev.atlassian.net/browse/PMM-13471) to add content to it